### PR TITLE
Adapt tests to work on SUSE

### PR DESCRIPTION
**Pull request by eliroca**

### CODE REVIEW

File: test/TEST-21-SYSUSERS/test.sh [hunk 1]
```diff
@@ -22,8 +22,9 @@ preprocess() {
     # see meson.build how to extract this. gcc -E was used before to
     # get this value from config.h, however the autopkgtest fails with
     # it
-    SYSTEM_UID_MAX=$(awk 'BEGIN { uid=999 } /^\s*SYS_UID_MAX\s+/ { uid=$2 } END { print uid }' /etc/login.defs)
-    SYSTEM_GID_MAX=$(awk 'BEGIN { gid=999 } /^\s*SYS_GID_MAX\s+/ { gid=$2 } END { print gid }' /etc/login.defs)
+    [[ -e /etc/login.defs ]] && login_defs_file="/etc/login.defs" || login_defs_file="/usr/etc/login.defs"
```
Comment (keszybz) [inline test/TEST-21-SYSUSERS/test.sh:25]: That seems reasonable for all distros.

File: test/create-busybox-container [hunk 2]
```diff
@@ -9,6 +9,16 @@ mkdir -p "$root"
 mkdir "$root/bin"
 cp $(type -P busybox) "$root/bin"
 
+ID_LIKE=$(awk -F= '$1=="ID_LIKE" { print $2 ;}' /etc/os-release)
+if [[ "$ID_LIKE" = *"suse"* ]]; then
+    mkdir -p "$root/lib"
+    mkdir -p "$root/lib64"
```
Comment (keszybz) [inline test/create-busybox-container:15]: Does suse really not do usr-merge?

Comment (keszybz) [reply test/create-busybox-container:15]: https://en.opensuse.org/openSUSE:Usr_merge says that opensuse does...

File: test/TEST-21-SYSUSERS/test.sh (hunk 1 above)
Comment (eliroca) [reply test/TEST-21-SYSUSERS/test.sh:25]: True.

File: test/create-busybox-container (hunk 2 above)
Comment (tblume) [reply test/create-busybox-container:15]: > Does suse really not do usr-merge?

That is work in progress but still not finished.
Some shared libraries, needed in the container, still reside below /lib for 32bit and /lib64 for 64bit.

Comment (keszybz) [reply test/create-busybox-container:15]: I think it's one of those case where "ripping off the band-aid" is the way to go... A brief moment of pain is better than dragging this out and dealing with workarounds for workarounds.

**Pull request merged**